helm-charts/konk/scripts/provision.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -xe
 
+secret_not_found () {
+  if [ "$(kubectl -n $NAMESPACE get secret $1 --ignore-not-found 2>&1 )" = "" ]
+  then
+    return 0
+  else
+    return 1
+  fi
```

Comment:
We can make this a little better. Right now, it's short and sweet but completely hides errors. In the function, we have the opportunity to be more verbose even though it wasn't feasible as a one-liner.

```suggestion
  output=$(kubectl -n $NAMESPACE get secret $1 --ignore-not-found 2>&1 )
  echo "$output"
  [ "$output" = "" ]
```
BTW, `return` is implicit in my example:
https://www.gnu.org/software/bash/manual/bash.html#Shell-Functions
> otherwise the function’s return status is the exit status of the last command executed